superset/config.py

```diff
@@ -471,8 +471,46 @@ def _try_json_readsha(filepath: str, length: int) -> Optional[str]:
     "DRILL_TO_DETAIL": False,
     "DATAPANEL_CLOSED_BY_DEFAULT": False,
     "HORIZONTAL_FILTER_BAR": False,
+    # Allow users to enable ssh tunneling when creating a DB.
+    # Users must check whether the DB engine supports SSH Tunnels
+    # otherwise enabling this flag won't have any effect on the DB.
+    "SSH_TUNNELING": False,
 }
 
+# ------------------------------
+# SSH Tunnel
+# ------------------------------
+# Allow users to set the host used when connecting to the SSH Tunnel
+# as localhost and any other alias (0.0.0.0)
+# ----------------------------------------------------------------------
+#                             |
+# -------------+              |    +----------+
+#     LOCAL    |              |    |  REMOTE  | :22 SSH
+#     CLIENT   | <== SSH ========> |  SERVER  | :8080 web service
+# -------------+              |    +----------+
+#                             |
+#                          FIREWALL (only port 22 is open)
+
+# ----------------------------------------------------------------------
+# class SSHManager:
+#      def validate(self, ssh_tunnel_params: Dict[str, Any]) -> None:
+#          # validation on CREATE + UPDATE on SSHTunnel Model
+#          # to block a request this function most raise an exception
+#          raise NotImplemented()
+
+#      def mutator(self, ssh_tunnel_params: Dict[str, Any]) -> Dict[str, Any]:
+#          # override any ssh tunnel configuration object
+#          raise NotImplemented()
```

Comment:
do we need to make these two methods required? They seems more like nice to haves.